<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Posit Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../TFG/iconos/positTool.png</normaloff>../../../TFG/iconos/positTool.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QToolButton" name="conversionButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>121</width>
        <height>118</height>
       </rect>
      </property>
      <property name="text">
       <string>Conversion to Posit</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../TFG/iconos/prueba conversion.png</normaloff>../../../TFG/iconos/prueba conversion.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="infoButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>121</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Posit Info</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>images/prueba info.png</normaloff>images/prueba info.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="multDivButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>270</y>
        <width>121</width>
        <height>88</height>
       </rect>
      </property>
      <property name="text">
       <string>Multiplication/Division</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../TFG/iconos/prueba multiplicacion.png</normaloff>../../../TFG/iconos/prueba multiplicacion.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="additionButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>121</width>
        <height>88</height>
       </rect>
      </property>
      <property name="text">
       <string>Addition/Subtraction</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../TFG/iconos/prueba suma.png</normaloff>../../../TFG/iconos/prueba suma.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="glossaryButton">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>0</y>
        <width>16</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>49</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Posit Info</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>230</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Conversion to Posit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>370</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Addition/Subtraction</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>370</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Multiplication/Division</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>GO</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>520</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>571</width>
        <height>371</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>569</width>
         <height>369</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="glossaryButton_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>0</y>
        <width>16</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>i</string>
      </property>
     </widget>
     <widget class="QComboBox" name="nBitsBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="esBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>n =</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>es =</string>
      </property>
     </widget>
     <widget class="QLabel" name="operationTitle">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>571</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
